docs/FAQs.md

```diff
@@ -0,0 +1,10 @@
+- [Frequently Asked Questions](#frequently-asked-questions)
+  - [How is Hive different than installer?](#how-is-hive-different-than-installer)
+
+# Frequently Asked Questions
+
+## How is Hive different than installer?
```

Comment:
Should probably substitute installer with "OpenShift Installer" everywhere it's used.